modules/kubernetes-addons/cilium/locals.tf

```diff
@@ -0,0 +1,68 @@
+locals {
+  name = "cilium"
+  default_helm_config = {
+    name        = local.name
+    chart       = local.name
+    repository  = "https://helm.cilium.io/"
+    version     = "1.12.1"
+    namespace   = "kube-system"
+    description = "cilium helm Chart deployment configuration"
+  }
+
+  set_values_vpc_cni = [
```

Comment:
lets move these to use values so that its easier for folks to choose between chaining or not and to extend the default of chaining mode via set values. you can remove the `set_values_default_cni` and the conditional logic entirely. 

We'll keep it simple here and just default to chaining on top of the AWS VPC CNI, and if users want to fully replace the CNI with cillium then thats a more involved process and they can override the values used by passing their own values

Reply:
Sure, that make sense!!